### PR TITLE
19457 sequences

### LINKED ISSUE #19457: Adding new Examples in the example package
Adding more examples will help people in understanding more about the Sympy Library.

### PR DESCRIPTION
**Pull request by Abhishek-IOT**
#### References to other Issues or PRs
Fixes #19457

#### Brief description of what is fixed or changed
Just made a few more examples in the examples package so that the beginner open source coder can understand more about it. Split the main method to two different method Normal Equation method and For Squared Equation Example

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by Abhishek-IOT** [comment]
This is open for review now

File: examples/all.py [hunk 1]
```diff
@@ -166,6 +166,7 @@ def run_example(example, *, reporter=None):
         mod = load_example_module(example)
         if reporter:
             suppress_output(mod.main)
+
```
Comment (friyaz) [inline examples/all.py:169]: I don't think this empty line is needed.

Comment (Abhishek-IOT) [reply examples/all.py]: deleted that line to resolve that issue

**Comment by czgdp1807** [comment]
Closing this for now due to inactivity and non-critical changes. Thanks for your contributions.

**Pull request closed without merging**